.circleci/integration/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+gem install pkg/fluent-plugin-splunk-hec-1.0.1.gem
+sudo mkdir /etc/fluent
+sudo cp .circleci/integration/fluent.conf /etc/fluent/fluent.conf
+sudo cp .circleci/integration/test_file.txt /etc/
+nohup fluentd & > /dev/null 2>&1
```

Comment:
i dont think you should redirect stdout,stderr to /dev/null, should be in a log file or stdout,stderr